pkg/virt-operator/resource/apply/reconcile.go

```diff
@@ -85,6 +85,27 @@ func objectMatchesVersion(objectMeta *metav1.ObjectMeta, version, imageRegistry,
 }
 
 func injectOperatorMetadata(kv *v1.KubeVirt, objectMeta *metav1.ObjectMeta, version string, imageRegistry string, id string, injectCustomizationMetadata bool) {
+	injectBaseOperatorMetadata(kv, objectMeta, version, imageRegistry)
+	objectMeta.Annotations[v1.InstallStrategyIdentifierAnnotation] = id
+	if injectCustomizationMetadata {
```

Comment:
Is this if branch still required? Is this func called somewhere with `injectCustomizationMetadata == false`?